spec/features/idv/hybrid_mobile/hybrid_mobile_spec.rb

```diff
@@ -151,7 +151,7 @@
       )
     end
 
-    it 'shows capture complete on mobile and error page on desktop', js: true do
+    it 'does not rate limit on last attempt if successful', js: true do
       user = nil
```

Comment:
This is a non-blocking nit.  

If you want to make this slightly more concise, you could delete line 155 and change line 158 from `user = sign_in_and_2fa_user` to `sign_in_and_2fa_user`.  The test still passed locally when I made the change.  I don't think you need to declare `user` as a variable.

Let me know if I'm missing anything here.

Reply:
Great idea! That was a result of copy/paste, and I found there were a few other tests in the file with the same issue. Fixed in d147bee0d9df829865aaa3dcf28676fb5404398c.